src/main/kotlin/com/sexysisters/tojserverv2/domain/wiki/domain/Wiki.kt

```diff
@@ -7,36 +7,47 @@ import javax.persistence.*
 @Entity
 @Table(name = "tbl_wiki")
 class Wiki(
-    @Embedded val name: Name
+    @Embedded
+    val name: Name
 ) : BaseTimeEntity() {
 
-    @Embedded var html: HTML = HTML()
-    @Embedded var markdown: Markdown = Markdown()
-    @Embedded var views: Views = Views()
+    @Embedded
+    var html: HTML = HTML()
+        private set
+
+    @Embedded
+    var markdown: Markdown = Markdown()
+        private set
+
+    @Embedded
+    var views: Views = Views()
+        private set
 
     @OneToOne(mappedBy = "wiki", fetch = FetchType.LAZY)
     var school: School? = null
+        private set
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     val id: Long = 0L
 
+    fun makeRelation(school: School) {
+        this.school = school
+        school.wiki = this
+    }
+
+    fun update(html: HTML, markdown: Markdown) {
+        this.html = html
+        this.markdown = markdown
+    }
+
+    fun countViews() {
+        val calculatedValue = viewsValue() + 1
+        this.views = Views(calculatedValue)
```

Comment:
Views 안에 있는 실제 값에 올려주는 건 어떤가요? 그럼 좀 더 객체지향적인 코드가 될 것 같습니다! 그리고 필요없는 코드도 줄어들어요.
```suggestion
        views.addCount();
```

Views.java
```java
fun addCount() = value + 1
```

Reply:
내부에 함수 만들어줘도 되나요?

Reply:
당연하죠!